app/javascript/components/App.js

```diff
@@ -1,12 +1,68 @@
-import React from "react";
+import React, { useEffect, useState } from "react";
 import PropTypes from "prop-types";
 import { BrowserRouter, Routes, Route } from "react-router-dom";
-class App extends React.Component {
-  render() {
-    return <React.Fragment>
-      
-    </React.Fragment>;
-  }
-}
+import Footer from "./components/Footer"
+import Header from "./components/Header"
+import YourFriends from "./pages/YourFriends";
+import AboutUs from "./pages/AboutUs";
+import LandingPage from "./pages/LandingPage"
+import MyProfile  from ".pages/MyProfile"
+import NewTask from ".pages/NewTask"
+import ProtectedIndex from "./pages/ProtectedIndex";
+import UpdateTask from "./pages/UpdateTask"
+import YourFriends from "./pages/YourFriends";
+import { render } from "react-dom";
+import UserShow from "./pages/UserShow";
+import UpdateUser from "./pages/UpdateUser";
+import Notfound from "./pages/Notfound";
+ 
+
+const App = (props) => {
+  const [users, setUsers] = useState([]);
+  useEffect(() => readUsers(), []);
+  useEffect(() => readTasks(), []);
```

Comment:
readUsers and readTasks are not defined. Either define them here in App.js or delete these lines